### DISCUSSION TITLE
Is BootstrapVue support Vue3 ?

### DISCUSSION THREAD

**Discussion by otsabi94** [bootstrap-vue/bootstrap-vue#6910] [Q&A] [answered]
Hello i'am trying to use BootstrapVue in my Vue App with vue 3 should i switch to vue 2 ???

**Comment by Hiws** [comment, marked as answer]
It does not.
Refer to: https://github.com/bootstrap-vue/bootstrap-vue/issues/5196 and https://github.com/bootstrap-vue/bootstrap-vue/issues/6872 for more information

**Reply by otsabi94** [reply]
tanks a lot
